lib/internal/checkUrl.js

```diff
@@ -24,6 +24,25 @@ function checkUrl(link, baseUrl, options, callback)
 		callHandler.async(callback, link);
 		return;
 	}
+
+	if ( cache !== undefined && cache !== null )
```

Comment:
```js
if (cache != null)
```
will check `undefined` and `null`, which are the first two in the list of non-strict equal conditions